ansible/templates/usb/scripts/headnode.sh.j2

```diff
@@ -191,6 +191,7 @@ function _build_vmmanifest() {
 				-e "s|@MON_IP@|${MON_IP}|" \
 				-e "s|@ROOT_AUTHORIZED_KEYS@|${ROOT_AUTHORIZED_KEYS}|" \
 				-e "s|@PGSQL_PDNS_PASSWORD@|${PGSQL_PDNS_PASSWORD}|" \
+				-e "s|@PDNS_RECURSOR_FORWARDERS@|.=${DNS_RESOLVERS/,/;}|" \
```

Comment:
We really should set this mdata conditionally. We don't want to flood (default) Google DNS servers by every recursive query in the cloud. By default, our pdns server should take care of all recursive queries by itself (opposite to asking a "smarthost"). 
Please add (in pseudo code):
```if recursors != 8.8.[48].[48]; then set metadata...```